NineChronicles.DataProvider/RenderSubscriber.cs

```diff
@@ -1,3 +1,5 @@
+using System.Reactive;
```

Comment:
Is this `using` needed?

Reply:
I must've left it when trying other things 😅. It's been removed now, thank you for checking!